packages/google_maps_flutter/google_maps_flutter/README.md

```diff
@@ -170,3 +170,11 @@ class MapSampleState extends State<MapSample> {
 ```
 
 See the `example` directory for a complete sample app.
+
+## Support
+
+This package is offered via an open source license. It is not governed by the Google Maps Platform Support [Technical Support Services Guidelines](https://cloud.google.com/maps-platform/terms/tssg), the [SLA](https://cloud.google.com/maps-platform/terms/sla), or the [Deprecation Policy](https://cloud.google.com/maps-platform/terms) (however, any Google Maps Platform services used by the library remain subject to the Google Maps Platform Terms of Service).
+
+This package adheres to [semantic versioning](https://semver.org/) to indicate when backwards-incompatible changes are introduced. Accordingly, while the library is in version 0.x, backwards-incompatible changes may be introduced at any time.
```

Comment:
All of our packages uses semver, as does essentially the entire Dart package ecosystem, so we don't say this at a per-package level.